commit 832ba8f1dd32f641eaf6504f71419f0dba19a8b5
Author: Bernhard Rosenkränzer <bero@lindev.ch>
Date:   Sun Feb 22 11:34:35 2015 +0100

    Fix typo in generated desktop files
    
    Desktop files generated by e.g. entering a custom application at
    the mimeopen -d prompt would get a "NoDsiplay" (rather than
    "NoDisplay") entry.
    
    Signed-off-by: Bernhard Rosenkränzer <bero@lindev.ch>

diff --git a/lib/File/MimeInfo/Applications.pm b/lib/File/MimeInfo/Applications.pm
index 40a57dc..af6e071 100644
--- a/lib/File/MimeInfo/Applications.pm
+++ b/lib/File/MimeInfo/Applications.pm
@@ -72,7 +72,7 @@ sub mime_applications_set_custom {
 	$object->set(
 		Type      => 'Application',
 		Name      => $word,
-		NoDsiplay => 'true',
+		NoDisplay => 'true',
 		Exec      => $command,
 	);
 	my (undef, undef, $df) = File::Spec->splitpath($desktop_file);
